src/vision/include/HSVFilterNode.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: November 11th, 2017
+ * Description: Performs an HSV filter on incoming data and republishes it
+ */
+
+#ifndef PROJECT_HSV_H
+#define PROJECT_HSV_H
```

Comment:
See comment in `HSVFilter.h` on preprocessor guards.

Reply:
done